ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -0,0 +1,348 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    os.environ["GOOGLE_CLOUD_PROJECT"] = MY_PROJECT
</pre><p>Is this required?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/377/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Yes